Vagrantfile

```diff
@@ -10,78 +10,26 @@ Vagrant.configure("2") do |config|
   # Disable update checks – we don't want people to redownload boxes
   config.vm.box_check_update = false
 
+  config.vm.provider "virtualbox" do |vb|
+    vb.memory = "1536"
+
+    # Downloads speeds within VMs are horrendous without these.
+    vb.customize ["modifyvm", :id, "--natdnshostresolver1", "on"]
+    vb.customize ["modifyvm", :id, "--natdnsproxy1", "on"]
+    vb.customize ["modifyvm", :id, "--nictype1", "virtio"]
```

Comment:
Didn't know about these settings, thanks!  (And in any case, the VM was able to saturate my DSL connection, so probably not much I can save.)

Reply:
Haha, yep, my VM was doing like 70 kb/s max, and I knew development couldn't continue like that. I just curated these after looking for them online.